docs/src/appendix/symmetric_tutorial.md

```diff
@@ -0,0 +1,1291 @@
+on a simple `TensorMap`.
+
+```@example symmetric_tutorial
+V = Vect[FermionParity](0 => 1, 1 => 1)
+t = ones(ComplexF64, V ← V ⊗ V)
+subblocks(t)
+```
+
+```@example symmetric_tutorial
+tp = permute(t, ((1,), (3, 2)))
+subblocks(tp)
+```
+In other words, when exchanging the two domain vector spaces, the reduced tensor elements of
+the `TensorMap` for which both uncoupled irreps in the domain of the corresponding fusion
+tree are odd picks up a minus sign, exactly as we would expect for fermionic charges.
+
+
+### Constructing the Hamiltonian
+
+We can directly construct the Hamiltonian terms as symmetric `TensorMap`s using the same
+procedure as before starting from their matrix elements in the occupation number basis.
+However, in this case we should be a bit more careful about the precise definition of the
+basis states in composite systems. Indeed, the tensor product structure of fermionic systems
+is inherently tricky to deal with, and should ideally be treated in the context of
+[*super vector spaces*](https://en.wikipedia.org/wiki/Super_vector_space). For two sites, we
+can define the following basis states on top of the fermionic vacuuum $\ket{00}$:
+```math
+\begin{align*}
+\ket{01} &= c_2^+ \ket{00}, \\
+\ket{10} &= c_1^+ \ket{00}, \\
+\ket{11} &= c_1^+ c_2^+ \ket{00}. \\
+\end{align*}
+```
+This definition in combination with the anticommutation relations above give rise to the
+nonzero matrix elements
+```math
+\begin{align*}
+c_1^+ c_2^- \ket{0, 1} &= \ket{1, 0}, \\
+c_1^- c_2^+ \ket{1, 0} &= - \ket{0, 1}, \\
+c_1^+ c_2^+ \ket{0, 0} &= \ket{1, 1}, \\
+c_1^- c_2^- \ket{1, 1} &= - \ket{0, 0}, \\
+N \ket{n} &= n \ket{n}.
+\end{align*}
+```
+While the signs in these expressions may seem a little unintuitive at first sight, they are
+essential to the fermionic nature of the system. Indeed, if we for example work out the
+matrix element of $c_1^- c_2^+$ we find
+```math
+\begin{align*}
+c_1^- c_2^+ \ket{1, 0} = c_1^- c_2^+ c_1^+ \ket{0, 0} = - c_2^+ c_1^- c_1^+ \ket{0, 0} = - c_2^+ (\mathbb{1} - c_1^+ c_1^-) \ket{0, 0} = - c_2^+ \ket{0, 0} = - \ket{0, 1}. \\
+\end{align*}
+```
+
+Once we have these matrix elements the hard part is done, and we can straightforwardly
+associate these to the following $f\mathbb{Z}_2$ fusion trees with corresponding reduced
+tensor elements,
+
+```@raw html
+<center><img src="../img/symmetric_tutorial/fZ2_fusiontrees.svg" alt="fZ2_fusiontrees" class="color-invertible" style="zoom: 170%"/></center>
+```
+
+Given this information, we can go through the same procedure again to construct $c^+ c^-$,
+$c^- c^+$ and $N$ operators as `TensorMap`s over $f\mathbb{Z}_2$-graded vector spaces.
+
+```@example symmetric_tutorial
+V = Vect[FermionParity](0 => 1, 1 => 1)
+```
+
+```@example symmetric_tutorial
+c⁺c⁻ = zeros(ComplexF64, V ⊗ V ← V ⊗ V)
+odd = FermionParity(1)
+for (s, f) in fusiontrees(c⁺c⁻)
+    if s.uncoupled[1] == odd && f.uncoupled[2] == odd && f.coupled == odd
+        c⁺c⁻[s, f] .= 1
+    end
+end
+c⁺c⁻
```

Comment:
Here and below, whatever we decide regarding the printing of `TensorMap`s (I agree with the general sentiment that the current printing output is not optimal), you probably intended to show the subblocks?
```suggestion
subblocks(c⁺c⁻)
```